lib/Red/Column.pm6

```diff
@@ -12,9 +12,8 @@ has Bool        $.auto-increment   = False;
 has             &.references;
 has             &!actual-references;
 has             $!ref;
-has Bool        $.nullable         = $!attr.package.^default-nullable;
+has Bool        $.nullable         = $!attr.package.HOW.?default-nullable($!attr.package) // False;
```

Comment:
Hah, that answers a thought I had the other day whether ```$foo.?^bar``` works :-)

Reply:
I’ve tried the `$foo.?^bar`... yes... doesn’t work... :(